framework/tests/all-systems/AssignmentContext.java

```diff
@@ -5,7 +5,10 @@
 
     void foo(String[] a) {}
 
-    @SuppressWarnings("determinism:invalid.type.on.conditional")
+    @SuppressWarnings({
+        "determinism:invalid.type.on.conditional",
+        "determinism:invalid.array.component.type"
+    }) // temporarily incorrect defaults for local variables
```

Comment:
Isn't this fixed now?

Reply:
It looks like there was an error before on the declaration. It looks like there's still an error but it's on the constructor instead. I updated the comment to reflect this.